targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/TOOLCHAIN_ARM_STD/rtl8195a.sct

```diff
@@ -55,7 +55,7 @@ LR_TCM 0x1FFF0000 0x10000 {
     }
 }
 
-LR_DRAM 0x30000000 0x200000 {
+LR_IROM1 0x30000000 0x200000 {
 
```

Comment:
These two regions had reminded as one region. Are there any reasons? LR_DRAM is actually not a region that only has text. I wonder the name change may cause further issues,

Reply:
@davidsaada

Reply:
Any suggestions?